scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,320 @@
+# -*- coding: utf-8 -*-
+"""
+Computing ihMT and non-ihMT maps
+Make a more details description !
+"""
+
+import argparse
+import os
+
+import glob
+import json
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_output_dirs_exist_and_empty)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument("in_folder",
+                   help='Path to the subject ihMT maps folder')
```

Comment:
You can give a short description here 6 images per echo with these names altpn ...
Maybe, maybe Max or somebody will ask you to split this into 6 inputs. That's also maybe a good option.
You give each echoes of each image with `nargs='+'` then you can easily check all input data (nii.gz and json files). 

That could be more explicit. What's your call on this?